content/guides/cli/scripts/index.md

```diff
@@ -22,10 +22,12 @@ and customize.
 * [Deprovision users and archive folders][2]
 * [Manage groups and collaborations][3]
 * [Report inactive users][4]
-* [Update user zones][5]
+* [Extract metadata][5]
+* [Update user zones][6]
 
 [1]: g://cli/quick-start/powershell-script-templates
 [2]: g://cli/scripts/deprovision-users
 [3]: g://cli/scripts/manage-groups-collaborations
 [4]: g://cli/scripts/report-inactive-users
-[5]: g://cli/scripts/user-zones-mass-update
\ No newline at end of file
+[5]: g://cli/scripts/extract-metadata
```

Comment:
I think you need to update this part, since the file name changed
`g://cli/scripts/metadata-extraction`